src/main/java/io/vertx/rabbitmq/impl/Utils.java

```diff
@@ -65,6 +71,26 @@ public static JsonObject toJson(AMQP.BasicProperties properties) {
     return json;
   }
 
```

Comment:
Could you add the javadoc ?

Reply:
Hi what is the current of the issue? I'm facing the same issue and looking forward to the new version including the patch. If only the javadoc is preventing the merge I could provide something ;).

Reply:
I believe there should be a test as well

Reply:
but well @waltersve any contribution is welcome :-)

Reply:
Hi @vietj, I created a pull-request including the patch, javadoc and tests. I hope that work for you.